### DISCUSSION TITLE
DynamicForm 2.4.4.5 released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4549] [Show and tell]
script loading fixes, refactoring for the DynamicForm Toolbox Control SDK.
Download now from the Oqtane Marketplace.
![image](https://github.com/user-attachments/assets/c5b82a35-f550-4284-afd5-515eda44e0ad)

**Comment by ADefWebserver** [comment]
link?

**Reply by leigh-pointer** [reply]
Download now from the Oqtane Marketplace.
